requirements.txt

```diff
@@ -1,9 +1,9 @@
 numpy
 pacfish
 googledrivedownloader
-torch>=1.11.0
+torch
 pytest
 matplotlib
 imageio
 scipy
-simpa
```

Comment:
We need the simpa requirement for the `data_generation` part of the project.
Do you think we should define separate requirement lists for each of the partial projects?
Or do you maybe have a different idea as well?